File: Modul 6/Soal 1/PRAK601-2410817210019-AchmadReihanAlfaiz.py
baris, kolom = map(int, input().split())
matrix = []

values = list(map(int, input().split()))

for i in range(baris):
    row = values[i * kolom : (i + 1) * kolom]
    matrix.append(row)


for row in matrix:
    print(" ".join(map(str, row)))

File: Modul 6/Soal 1/PRAK601-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int row, column;
     scanf("%d %d", &row, &column);

     int matrix[row][column];

     for (int i = 0; i < row; i++)
     {
          for (int j = 0; j < column; j++)
          {
               scanf("%d", &matrix[i][j]);
               printf("%d ", matrix[i][j]);
          }
          printf("\n");
     }

     return 0;
}